docs/additional-functionality/shuffle-docker-examples/Dockerfile.rocky_no_rdma

```diff
@@ -33,11 +33,11 @@ ARG UCX_VER
 ARG UCX_CUDA_VER
 ARG UCX_ARCH
 
-RUN yum update -y && yum install -y wget bzip2 numactl-libs libgomp
+# note that libibmad is a temporary workaround for a missed dependency in ucx-1.16 rpm
+RUN yum update -y && yum install -y wget bzip2 numactl-libs libgomp libibmad
 RUN ls /usr/lib
 RUN mkdir /tmp/ucx_install && cd /tmp/ucx_install && \
   wget https://github.com/openucx/ucx/releases/download/v$UCX_VER/ucx-$UCX_VER-centos8-mofed5-cuda$UCX_CUDA_VER-$UCX_ARCH.tar.bz2 && \
   tar -xvf *.bz2 && \
-  rpm -i ucx-$UCX_VER*.rpm && \
-  rpm -i ucx-cuda-$UCX_VER*.rpm --nodeps && \
+  rpm -i `ls ucx-[0-9]*.rpm ucx-cuda-[0-9]*.rpm` --nodeps && \
```

Comment:
So the rpm is not named consistently with the version?

Reply:
This is correct :( It is an RC issue as well, when you drop the RC it works. That said, I figured this was a bit more robust, because we have nothing else in this directory `/tmp/ucx_install`, so it's only UCX at this rate.